tests/pywrap-opt-sowreap/input.dat

```diff
@@ -11,4 +11,4 @@ set {
    geom_maxiter 6
 }
 
-optimize('mp3',mode='soW',dertype='energy')
+optimize('mp3',mode='sow',dertype='energy')
```

Comment:
Did this not work with the awkward capitalization? If not, then i'd rather fix the kwargs case handling.